lib/Handler/CompletionHandler.php

```diff
@@ -80,7 +88,7 @@ public function completion(TextDocumentItem $textDocument, Position $position):
         foreach ($suggestions as $suggestion) {
             /** @var Suggestion $suggestion */
             $completionList->items[] = new CompletionItem(
-                $suggestion->name(),
+                $this->suggestionLabelFormatter->format($suggestion),
```

Comment:
hmm, why do we replace `name` with `label`? Should this be `name` formatter?

Reply:
Thanks for attention. It came from the first trying and was left untouched, even when moved to formatter ;)

Indeed, for example in `WorseClassMemberCompletor` a method name is mapped to suggestion name. I am surprise how then version using label works as expected.